docs/sphinx/source/whatsnew/v0.9.2.rst

```diff
@@ -14,7 +14,7 @@ Bug fixes
 * :py:func:`pvlib.irradiance.get_total_irradiance` and
   :py:func:`pvlib.solarposition.spa_python` now raise an error instead
   of silently ignoring unknown parameters (:ghpull:`1437`)
-  
+* Updated version of numba in asv.conf from 0.36.1 to 0.40.0 to solve numba/numpy conflict. (:issue:`1439`, :pull:`1440`)
```

Comment:
Let's move this to a new `Benchmarking` section instead of `Bug fixes`

Reply:
Sound good! I'll update it